src/cartservice/otel-agent-config.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+otel-agent-config.yaml: |
```

Comment:
I'm a bit confused about this extra file. Isn't the config already baked into the k8s manifests?

Reply:
Yes. An artifact left over from some early debugging.  I removed it.